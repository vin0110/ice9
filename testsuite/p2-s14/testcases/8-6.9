#Test case: 8:2
forward check (a: int, st: string) : int;

var a: int;

proc check(a: int, bo: bool, st: string) : int
    var b: int; 
    fa i := 1 to 10 -> 
        write b + a; 
    af 
    check = a; 
end

a := check(100,true,"Hello");
