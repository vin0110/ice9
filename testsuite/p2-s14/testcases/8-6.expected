line 6: Semantic error: Proc does not match forward definition
